functions/src/pubsub/schedule/sync-mutualaidhub.ts

```diff
@@ -0,0 +1,170 @@
+import * as firebase from 'firebase';
+import { CollectionReference, FieldPath, GeoPoint, QueryDocumentSnapshot } from '@google-cloud/firestore';
+import * as functions from 'firebase-functions';
+
+import { db } from '../../app';
+
+import { beginningStats } from './sync-util';
+
+import { ContactGroup, MARKER_COLLECTION_ID, MarkerInfo } from '../../models/markers';
+import isEqual = require('lodash/isEqual');
+import merge = require('lodash/merge');
+
+// Define the fields we're filtering on as constants here to ensure that they
+// can be used in the Source type, and we get a type error if we need to update
+// them.
+const FIELD_SOURCE = 'source';
+const FIELD_NAME = 'name';
+
+type Source = (MarkerInfo[typeof FIELD_SOURCE] & {})[typeof FIELD_NAME];
+
+const SOURCE_NAME_FIELD_PATH = new FieldPath(FIELD_SOURCE, FIELD_NAME);
+const SOURCE_NAME_FIELD_VALUE: Source = 'mutualaidhub.org';
+
+const MARKER_COLLECTION = db.collection(MARKER_COLLECTION_ID) as CollectionReference<MarkerInfo>;
+
+const MUTUAL_AID_HUB_APP_CONFIG = {
```

Comment:
These should be `config` variables from a maintenance perspective, but at the same time not a huge concern.

Reply:
Yeah.